lib/draft.ts

```diff
@@ -304,22 +280,36 @@ export class Dorm {
   into = this.table;
   
   /* ------------------------------ AS METHOD ------------------------------ */
-  as(arg:string, target:string){
-    this.callOrder.push('AS');
-    
-    if(this.info.join.length){
-      const joinList = this.info.join;
-    joinList.forEach(el => {
-      if(el.table === target) el.table = arg;
-      if(el.on && el.on.includes(target)) {
-        el.on.split(' ').forEach((word:string) =>  {
-          if(word === target) word = arg;
-        })
-      }
-    })
-    }
+  // as(arg:string, target:string){
```

Comment:
Hey myo, 

Make sure to work on As method in your different feature branch and only PR working code.  We don't want to put notworking/unrelated work on the production level codebase.